### PR TITLE
Change wording and add default phone number country on the committee page

### PR DESCRIPTION
**Pull request by illusionOfParadise**
The country by default on the committee form will be taken from local in another PR.

### CODE REVIEW

File: src/Form/CommitteeCommandType.php [hunk 1]
```diff
@@ -31,6 +31,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('address', AddressType::class)
             ->add('phone', PhoneNumberType::class, [
                 'widget' => PhoneNumberType::WIDGET_COUNTRY_CHOICE,
+                'preferred_country_choices' => ['FR'],
```
Comment (HeahDude) [inline src/Form/CommitteeCommandType.php]: Preferred !== preset

File: src/Form/TypeExtension/PhoneNumberTypeExtension.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        return PhoneNumberType::class;
+    }
+
+    public function setDefaultRegion(FormEvent $event)
+    {
+        $form = $event->getForm();
+        $defaultRegion = $form->getConfig()->getOption('default_region');
+
+        if (!$event->getData() && PhoneNumberUtil::UNKNOWN_REGION !== $defaultRegion) {
+            $form->get('country')->setData($defaultRegion);
```
Comment (HeahDude) [inline src/Form/TypeExtension/PhoneNumberTypeExtension.php]: `$event->setData(['country' => $defaultRegion, 'number' => ''])`

Comment (HeahDude) [reply src/Form/TypeExtension/PhoneNumberTypeExtension.php]: Hmm maybe even the `PhoneNumber` object instead.

File: src/Form/TypeExtension/PhoneNumberTypeExtension.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        return PhoneNumberType::class;
+    }
+
+    public function setDefaultRegion(FormEvent $event)
+    {
+        $form = $event->getForm();
+        $defaultRegion = $form->getConfig()->getOption('default_region');
+
+        if (!$event->getData() && PhoneNumberUtil::UNKNOWN_REGION !== $defaultRegion) {
+            $event->setData((new PhoneNumber())->setCountryCode(33)->setNationalNumber(''));
```
Comment (HeahDude) [inline src/Form/TypeExtension/PhoneNumberTypeExtension.php]: Hmm shounld't we do like https://github.com/EnMarche/en-marche.fr/pull/2111/files#diff-d4d5073bfd6cedfc2c729bbb3fe53b99R141?

**Comment by HeahDude** [comment]
Please don't forget to target `develop`.

File: src/Form/TypeExtension/PhoneNumberTypeExtension.php (hunk 3 above)
Comment (illusionOfParadise) [reply src/Form/TypeExtension/PhoneNumberTypeExtension.php]: I'll change this in the PR which uses the local to set default phone number country.

File: templates/committee/show.html.twig [hunk 4]
```diff
@@ -194,13 +194,12 @@
         <section>
             <div class="committee__waiting-for-approval">
                 Votre comité est en attente de validation par les équipes d'En Marche !
-                Vous serez alerté(e) par e-mail quand nous l'aurons accepté.
+                Vous serez alerté(e) par e-mail quand il sera validé.
+                <p>
+                    Une fois que votre comité sera validé, vous pourrez contacter les adhérents suivants votre comité
+                    sur cette page.
```
Comment (guillaumedenhez) [inline templates/committee/show.html.twig]: @HeahDude possible de remplacer le 2e paragraphe par "Une fois validé, vous pourrez contacter les adhérents suivant votre comité" ?

**Pull request merged**